GNUmakefile

```diff
@@ -39,7 +39,7 @@ chainlink: operator-ui ## Build the chainlink binary.
 	go build $(GOFLAGS) -o $@ ./core/
 
 .PHONY: chainlink-build
-chainlink-build:
+chainlink-build: ## Build & install the chainlink binary.
```

Comment:
Is this different than doing `go install` directly? Why do we manually reconstruct `GOPATH`/`GOBIN`?